compiler/qsc/src/bin/qsc.rs

```diff
@@ -74,12 +75,23 @@ fn main() -> miette::Result<ExitCode> {
         dependencies.push(store.insert(qsc::compile::std(&store, target)));
     }
 
-    let sources = cli
+    let mut sources = cli
         .sources
         .iter()
         .map(read_source)
         .collect::<miette::Result<Vec<_>>>()?;
 
+    if sources.is_empty() {
+        let fs = StdFs;
+        let manifest = Manifest::load()?;
```

Comment:
Typically you can provide the manifest file itself, or the directory containing it, as the argument to the compiler. Is the expectation here that you would always set current directory first and then that's all that is used to try to locate a manifest?